frontend/ui/widget/dictquicklookup.lua

```diff
@@ -726,10 +726,12 @@ function DictQuickLookup:init()
 
     -- NT: add dict_title.left_button and lookup_edit_button to FocusManager.
     -- It is better to add these two buttons into self.movable, but it is not a FocusManager.
-    -- Only self.button_table is a FocusManager, so workaground is inserting these two buttons into self.button_table.layout.
+    -- Only self.button_table is a FocusManager, so the workaround is inserting these two buttons into self.button_table.layout.
     if Device:hasDPad() then
-        table.insert(self.button_table.layout, 1, { self.dict_title.left_button });
-        table.insert(self.button_table.layout, 2, { lookup_edit_button });
+        table.insert(self.button_table.layout, 1, { self.dict_title.left_button })
```

Comment:
it appears that selecting this option with focus manager and <kbd>press</kbd>, and doing it by pressing <kbd>Menu</kbd> yields two different results, one is nice and orderly (FM one), the other other one is the same mess from before (Menu key bound event).

Reply:
<p align="center">
<img src="https://github.com/user-attachments/assets/1ced504e-3028-4c73-aefc-a12d2c88b356" width=40% height=40%>
<img src="https://github.com/user-attachments/assets/df10dae8-245e-4795-b9a9-77349a8fddd3" width=40% height=40%> 
</p>

Reply:
This stuff is still a bit rough and jagged, it acts really funny (not the hilarious kind) when using cursor keys. If you can try it on k4 it would be best cause i can’t even begin to explain how unpredictable it is.

Reply:
Is that widget used somewhere that *doesn't* require network connectivity?

Reply:
I don’t think you would get as many results anywhere else. Unless you have like 20 dictionaries that is.

Reply:
I think this explains the discrepancy between using Focus manager and pressing the menu button

https://github.com/koreader/koreader/blob/ab47bd10a4b65a7ad5161b80fbcbbbf03689c2aa/frontend/ui/widget/dictquicklookup.lua#L232-L237